lib/MwbExporter/Formatter/Doctrine2/Annotation/OnCorps/ApiPlatformFieldAnnotations.php

```diff
@@ -105,7 +106,7 @@ public function processFields(?string $comment = null): self
     public function generateAnnotationPropertyDetails(string $fieldName, ?string $typeKey = ""): string
     {
         $typeInfo = (object)$this->typeInfo[$typeKey];
-        $details = '"'.$fieldName.'"';
+        $details = '"'.$this->fields[$fieldName]->filterName.'"';
```

Comment:
Is this a future thing? Looks like ->filterName and key (name) share the same value

Reply:
For search  on fk fields name !== filterName for the @ApiFilter annotation. e.g. offering_family_id => offeringFamily